EDU-3012: Update Failure and Call Stack details to current Web UI (#315)
temporalio/temporal-learning: docs/getting_started/python/first_program_in_python/index.md

![The next Activity](images/activity_failure.png)

> 5. Click any ActivityTaskFailed event link to see Failure detail information and a Call Stack trace.

Reviewer: ```suggestion
5. Click any `ActivityTaskFailed` event link to see the Failure details and call stack.
```

Author: Thank you. Adopted.
